src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -58,7 +58,7 @@ public DateTime StartTime
                 // nanoseconds elapse from boot to that the process started) to seconds.
                 EnsureState(State.HaveId);
                 Interop.libproc.rusage_info_v3 info = Interop.libproc.proc_pid_rusage(_processId);
-                double seconds = info.ri_proc_start_abstime / NanoSecondToSecondFactor;
+                double seconds = info.ri_proc_start_abstime / (double)NanoSecondToSecondFactor;
```

Comment:
Thanks.  I started with that, but then figured it was better to keep the type as an integral representation, since it itself doesn't logically require floating-point.